lib/404.php

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>404 Not Found</title>
```

Comment:
On the 404 page we can also add the same CSS stylesheets as the `html` page.

```suggestion
	<title>404 Not Found</title>
	<meta content="width=device-width" name="viewport" />
	<link rel="shortcut icon" href="<?php echo EDITH_URI ?>/public/icon.png" />
	<link rel="stylesheet" href="<?php echo EDITH_URI ?>/public/html.css" />
```